### ISSUE TITLE
Terms of use text box not keyboard accessible

### ISSUE THREAD

**Issue by BryceStevenWilley** [SuffolkLITLab/docassemble-AssemblyLine#1061]
> The scrollable terms of use text box can be tabbed to in Chrome, but not all browsers support this by default. It needs "tabindex=0" and an accessible label for screen readers. https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=AxeChrome

> Add "tabindex=0" to the scrollable text box so it is focusable via keyboard in all browsers. Add a descriptive "aria-label" (such as "Terms of use") or use "aria-labelledby" to associate it with a visible heading so screen reader users understand what the region contains. Note: Firefox and Chrome already make scrollable regions keyboard accessible by default, with Safari being the remaining holdout (https://adrianroselli.com/2022/06/keyboard-only-scrolling-areas.html). Once Safari catches up, this tabindex may no longer be necessary.

(See A11Y-07)

**Comment by ekressmiller** [comment]
@nonprofittechy, thanks for your work on this!

If I'm understanding correctly, we should add a "subject" to our templates so that they will then be `aria-labelledby` that.  When I [add a "subject"](https://github.com/mplp/docassemble-mlhframework/pull/214/changes) it gets added to the top of the scrollable area as an H3, but H3 is not the appropriate heading level on our page (it comes right after an H1 so should be an H2). Is there a way to change the heading level?

<img width="1033" height="391" alt="Image" src="https://github.com/user-attachments/assets/4aa7ee52-df35-4d90-8672-a2adfaebb4f0" />
(The second heading in the image is an H2 that was already in the body of the template, which we would get rid of.)

**Comment by nonprofittechy** [comment]
Let me check. I might have set the wrong default, but I can also make it configurable!

**Comment by nonprofittechy** [comment]
You are right, h2 was also right level for us. But I made it configurable. See: https://github.com/SuffolkLITLab/docassemble-ALToolbox/pull/362

**Comment by ekressmiller** [comment]
Awesome, thank you!
